dev-libs/libfilezilla/libfilezilla-0.30.0.recipe

```diff
@@ -8,27 +8,33 @@ extremely efficient
 which simplifies dealing with timestamps originating from different sources
 - Simple process handling for spawning child processes with redirected I/O"
 HOMEPAGE="https://lib.filezilla-project.org/"
-COPYRIGHT="2015-2017 Tim Kosse"
+COPYRIGHT="2015-2021 Tim Kosse"
 LICENSE="GNU GPL v2"
 REVISION="1"
-SOURCE_URI="http://download.filezilla-project.org/libfilezilla/libfilezilla-$portVersion.tar.bz2"
-CHECKSUM_SHA256="ecbaa674c0ad0b63df842b8cde17935a497dd58c3749baa281c67cf5878e64f7"
+SOURCE_URI="https://download.filezilla-project.org/libfilezilla/libfilezilla-$portVersion.tar.bz2"
+CHECKSUM_SHA256="c16df6dacdb2ded4f6e05141b4681eda91a5a1ba052900a24a9f84e65a50dc40"
+PATCHES="libfilezilla-$portVersion.patchset"
 
 ARCHITECTURES="all ?x86_gcc2"
```

Comment:
Previous one hasn't been checked/enabled for gcc2, if it's not building there you could mark this as broken, otherwise LGTM (haven't checked a build here (yet)) :)

Reply:
Just strike this comment ;) On another bump we could look into it :) Thanks! :+1: